designs/digital_pll_sky130_fd_sc_hd/config.tcl

```diff
@@ -8,7 +8,7 @@ set ::env(VERILOG_FILES) $::env(DESIGN_DIR)/src/digital_pll.v
 set ::env(SYNTH_READ_BLACKBOX_LIB) 1
 
 # Fill this
-set ::env(CLOCK_PERIOD) "15.8"
+set ::env(CLOCK_PERIOD) "18"
```

Comment:
Are we free to mess with the clock period ?

Reply:
It's not passing otherwise.